hack/update-guard-service.sh

```diff
@@ -0,0 +1,2 @@
+export KO_DOCKER_REPO=us.icr.io/knat
```

Comment:
I'd take this out and note in the readme that users should set this variable on their systems